pkg/sessionctx/variable/sysvar.go

```diff
@@ -1480,6 +1480,11 @@ var defaultSysVars = []*SysVar{
 		s.SetStatusFlag(mysql.ServerStatusNoBackslashEscaped, sqlMode.HasNoBackslashEscapesMode())
 		return nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBLoadBindTimeout, Value: "200", Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxInt32, IsHintUpdatableVerfied: false, SetGlobal: func(ctx context.Context, vars *SessionVars, s string) error {
```

Comment:
Should confirm the variable name and default value with PM

Reply:
It has been confirmed by PM.